### PR TITLE
[docs] spark rename paimon table need notice can't put catalog name before the renamed-to table

### PR DESCRIPTION
**Pull request by leaves12138**
We can't put catalog name before the renamed-to table, it will throw an error

### CODE REVIEW

**Review by JingsongLi — CHANGES_REQUESTED** [review]
You can add this in `/how-to/altering-tables`.

File: docs/content/engines/spark3.md [hunk 1]
```diff
@@ -302,4 +302,4 @@ All Spark's data types are available in package `org.apache.spark.sql.types`.
 - Currently, Spark's field comment cannot be described under Flink CLI.
 - Conversion between Spark's `UserDefinedType` and Paimon's `UserDefinedType` is not supported.
 
-{{< /hint >}}
+{{< /hint >}}
```
Comment (JingsongLi) [inline docs/content/engines/spark3.md:305]: revert this?

Comment (leaves12138) [reply docs/content/engines/spark3.md]: done

File: docs/content/how-to/altering-tables.md [hunk 2]
```diff
@@ -80,9 +80,19 @@ ALTER TABLE my_table RENAME TO my_table_new;
 
 {{< tab "Spark3" >}}
 
+The simplest sql to call is:
 ```sql
 ALTER TABLE my_table RENAME TO my_table_new;
 ```
+{{< hint info >}}
```
Comment (JingsongLi) [inline docs/content/how-to/altering-tables.md:87]: remove `hint info`, such a long hint is not good

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**